app/libs/js/LEAF/formGrid.js

```diff
@@ -61,6 +60,107 @@ var LeafFormGrid = function (containerID, options) {
     showIndex = false;
   }
 
+  /**
+   * @param values (required) object of cells and names to generate grid
+   * @param showScriptTags (default false) whether to display script tags
+   * @memberOf LeafFormGrid
+   * Returns copy of values with cells property html entities decoded
+   */
+  function decodeCellHTMLEntities(values, showScriptTags = false) {
+    let gridInfo = { ...values };
+    if (gridInfo?.cells) {
```

Comment:
gridInfo.cells is not a boolean.  We are trying to make all our "if"s on actual boolean values to keep things future proof.